lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -989,6 +989,7 @@
     <!-- Flexible Image Transport System (FITS) used in Astronomy https://fits.gsfc.nasa.gov/ https://fits.gsfc.nasa.gov/rfc4047.txt -->
     <datatype extension="fits" type="galaxy.datatypes.binary:FITS" mimetype="application/octet-stream" display_in_upload="true" description="Flexible Image Transport System (FITS) used in Astronomy"/>
     <datatype extension="chain" type="galaxy.datatypes.chain:Chain" display_in_upload="true"/>
+    <datatype extension="net" type="galaxy.datatypes.chain:Net" display_in_upload="true"/>
```

Comment:
add the link as url or description as well to the upstream doc?

Reply:
thanks! resolved in d4aa9c4